.travis.yml

```diff
@@ -0,0 +1,64 @@
+#
+# Turn off email notifications entirely
+#
+notifications:
+  email: false
```

Comment:
Why not sending notifications? Jenkins does this quite well (only mails when the build breaks)

If only one e-mail address is allowed we could use some forward e-mail address, e.g. travis@libelektra.org